spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -67,13 +68,21 @@ def assert_specific_vm
     expect(vm.key_pairs.first.ems_ref).to_not be_nil
     expect(vm.key_pairs.first.ems_ref).to match(/^[-0-9a-z_]{1,64}/)
   end
-
+  
```

Comment:
Looks like some trailing whitespace made it in

Reply:
Yeah, those were introduced by the web editor when I fixed my merge problem. My IDE removed them when I had to fix the tests.

Reply:
Oh okay, yeah don't use the web editor for conflicts (see https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/177#issuecomment-809580010)